.github/workflows/build_and_push_nf_base_images.yml

```diff
@@ -0,0 +1,68 @@
+name: Push new nextflow images to ECR
+on:
+  workflow_dispatch:
+  push:
+    # branches: master
+    paths:
+      - 'nextflow-base-images/**' #Runs every time this folder gets updated
+      - '.github/workflows/build_and_push_nf_base_images.yml'
+  schedule:
+    - cron: '0 0 * * 6'  # Run at midnight UTC every Saturday
+
+jobs:
+  build_and_push_nf_images:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          path: containers
+
+
+      - name: Checkout other private repository
+        uses: actions/checkout@v3
+        with:
+          repository: uc-cdis/base-images
+          token: ${{ secrets.PLANXCYBORG_TOKEN }}
+          path: base-images
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+
+      - name: Login to Amazon ECR
+        env:
+          AWS_REGION: us-east-1  # Replace with your AWS region
+          AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID_ACCT_654654631253_ECR }}
+          AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY_ACCT_654654631253_ECR }}
+        run: |
+          aws ecr-public get-login-password --region $AWS_REGION | docker login --username AWS --password-stdin public.ecr.aws/u5x5h6w3
+
+      - name: Build and push Docker images
+        run: |
+              dir=base-images/amazonlinux-base/
+              image_name=nextflow-approved/public
+              tag_name=$(basename "$dir")
+              echo "$image_name:$tag_name"
+              docker build -t public.ecr.aws/u5x5h6w3/$image_name:$tag_name $dir
+              docker push public.ecr.aws/u5x5h6w3/$image_name:$tag_name
+
+              for dir in containers/nextflow-base-images/*/;do
+                  image_name=nextflow-approved/public
+                  tag_name=$(basename "$dir" | sed 's/_/./g')
+                  docker build -t public.ecr.aws/u5x5h6w3/$image_name:$tag_name $dir
+                  docker push public.ecr.aws/u5x5h6w3/$image_name:$tag_name
+                  echo "$image_name:$tag_name"
+                  echo $dir
+              done
+
+      - name: Trigger `get_layer_info_for_nf_imgs.yml` workflow
+        env:
+          GH_TOKEN: ${{ secrets.PLANXCYBORG_TOKEN }}
```

Comment:
I was suggesting moving the `get_layer_info_for_nf_imgs` steps to this workflow, not triggering it. This is fine too, but in that case i think we should remove the schedule from the `get_layer_info_for_nf_imgs` workflow?

Any reason you can think of for keeping the 2 workflows separate?

Reply:
I wanted to merge them into one as well since I did not see any use case in the current scenario where we would want to trigger the layers workflow in a standalone way. Since we already had the layers workflow developed, I thought this may add more flexibility in terms of triggering (if need be) and wouldn't add multiple responsibilities to one single workflow.

Reply:
ok in that case update the other wf to remove the other schedule, and maybe add a comment to say this wf triggers it